Makefile

```diff
@@ -7,21 +7,25 @@ gocache := $(shell go env GOCACHE)
 
 GOBIN ?= $(gopath)/bin
 
-generated_code_aws_sdk_mocks := $(wildcard pkg/eks/mocks/*API.go)
+
+always_generate_in_packages := ./pkg/nodebootstrap ./pkg/addons/default ./pkg/addons
 
 generated_code_deep_copy_helper := pkg/apis/eksctl.io/v1alpha5/zz_generated.deepcopy.go
 
-all_generated_code := \
-  pkg/addons/default/assets.go \
+generated_code_aws_sdk_mocks := $(wildcard pkg/eks/mocks/*API.go)
+
+conditionally_generated_files := \
+  site/content/usage/20-schema.md \
+  $(generated_code_deep_copy_helper) $(generated_code_aws_sdk_mocks)
```

Comment:
We need to always generate these files during `make build`  too since otherwise the build/tests will fail when something changes.

If anything, I see the "always_generate_in_packages" more optional than these. Why the separation now?

Reply:
Deep copy helpers and AWS SDK mocks take long time.

The markdown schema has relatively dependencies, albeit slightly wrong at present. I didn't want to touch it as part of this.

Reply:
> take long time.
How long? (I'd rather have something reliably long, than unreliably slightly less long. 😅 )

Reply:
I think mocks take on order of 30s-45s, and don't change most of the time.